### PR TITLE
Basestation interface

### PR DESCRIPTION
**Pull request by chynu**
Ready to test with GUI:
 - add bot
 - command bot
 - discover bot
 - send KV (needs to be tested with scripts on bot)

Left to finish:
 - scenarios
 - xbox

### CODE REVIEW

File: gui/main.py [hunk 1]
```diff
@@ -38,14 +49,78 @@ def make_app(self):
...
+        print(info)
+        print(discovered_bots)
+        print("That was the info.")
+
+        name = info['name']
+        ip = info['ip']
+        port = info['port']
+
+        bot_name = BaseStation().get_bot_manager().add_bot(name, ip, port)
+        print('THING')
```
Comment (leotrj) [inline gui/main.py]: Redundant Print statement

File: gui/static/js/interface.js [hunk 2]
```diff
@@ -84,11 +84,45 @@ $("#send").click(function(event) {
...
+            console.log("Adding is a success!");
+            console.log(data);
+            updateDropdown(true, data, data);
+        },
+        error: function(error) {
+            console.warn(error);
+            updateDropdown(true, error.responseText, error.responseText);
+        }
+    });
+    console.log("hello pls work");
```
Comment (leotrj) [inline gui/static/js/interface.js:123]: More redundant print statements

**Review by leotrj — CHANGES_REQUESTED** [review]

File: gui/main.py (hunk 1 above)
Comment (chynu) [reply gui/main.py]: Done.

File: gui/static/js/interface.js (hunk 2 above)
Comment (chynu) [reply gui/static/js/interface.js:123]: Done.

**Review by leotrj — APPROVED** [review]

**Pull request merged**